atlas/babel.config.json

```diff
@@ -4,6 +4,5 @@
 		"@babel/preset-react",
 		"@babel/preset-typescript",
 		"@emotion/babel-preset-css-prop"
-	],
-	"babelrcRoots": ["*", "packages/*"]
```

Comment:
Why was this removed?

Reply:
By mistake I think, restored it in 0fcbed9e3a61588cb8ab10868bd5b27439cc0b01